### PR TITLE
feat: Add confetti when win

### LINKED ISSUE #134: Add Confettii after a player wins
image

### PR DESCRIPTION
**Pull request by ssk090**
- Added confetti when win
- Fixes #134 

https://github.com/code100x/chess/assets/22127725/d70e6218-10d4-436d-a7f8-bcb2318571d9

### CODE REVIEW

**Comment by ssk090** [comment]
Hey @hkirat ,
Please check this PR, it adds confetti when a player wins

File: apps/frontend/src/screens/Game.tsx [hunk 1]
```diff
@@ -1,158 +1,193 @@
...
+      );
+    }
+  }, [chess, socket]);
 
-    return <div className="">
-        <div className="justify-center flex pt-4 text-white">
-            {gameMetadata?.blackPlayer?.name} vs {gameMetadata?.whitePlayer?.name}
-        </div>
-        {result && <div className="justify-center flex pt-4 text-white">
-            {result}    z
```
Comment (ssk090) [inline apps/frontend/src/screens/Game.tsx:131]: Removing this Letter 'z' from here, rest are whitespaces and lints

File: apps/frontend/src/components/ChessBoard.tsx [hunk 2]
```diff
@@ -1,275 +1,306 @@
...
-                    height={512}
-                    style={{ position: "absolute", top: 0, left: 0, pointerEvents: "none" }}
-                    onContextMenu={(e) => e.preventDefault()}
-                    onMouseDown={(e) => {
+                              if (moveResult) {
+                                if (moveResult.san.includes('#')) {
+                                  setGameOver(true);
+                                }
+                              }
+                            }
```
Comment (ssk090) [inline apps/frontend/src/components/ChessBoard.tsx:216]: These lines are for the confetti, rest are whitespaces and lints

**Comment by hkirat** [comment]
merge conflicts

**Pull request closed without merging**